src/Templates/NIPSWeb/import.twig

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
I'm wondering how easy it'd be to deduplicate the stuff that's been copy/pasted from the general timeslot selector. They both just ultimately spit out a timeslotid, but do something different with it, right?